experimental/Schemes/BlowupMorphism.jl

```diff
@@ -278,3 +278,101 @@ function maps_with_given_codomain(phi::CoveringMorphism, V::AbsSpec)
   return result
 end
 
+@Markdown.doc """
+  show_details(Bl::BlowupMorphism; more::Bool = false)
+
+For a `BlowupMorphism` ``p : Y → X`` display domain, codomain, center and exceptional divisor in a detailed view.
+# Examples
+```jldoctest
+julia> R, (x,y,z) = QQ["x", "y", "z"];
+
+julia> A3 = Spec(R);
+
+julia> I = ideal(R,[x,y,z]);
+
+julia> bl = blow_up(A3,I)
+BlowupMorphism{CoveredScheme{QQField}}(relative projective scheme over covered scheme with 1 affine patches in its default covering, covered scheme with 1 affine patches in its default covering, sheaf of ideals on covered scheme with 1 affine patches in its default covering, #undef, #undef, Oscar.EffectiveCartierDivisor{CoveredScheme{QQField}}(covered scheme with 3 affine patches in its default covering, sheaf of ideals on covered scheme with 3 affine patches in its default covering, Covering with 3 patches, #undef))
+
+julia> show_details(bl)
+Blow up of a Covered Scheme with 1 Charts at a 0-dimensional Center leading to a Covered Scheme with 3 Charts.
```

Comment:
It looks to me like BlowupMorphism is missing a `Base.show` method. Could you please provide it? 
Its output should be similar to the output of `show_details(bl,more=false)`. (If the dimension of the center is not known, we can omit it and not trigger its computation)

Reply:
It seems to me that your show details functions (with `more=false`) should rather be the way we print stuff, because surely they are more useful than what we have.

Reply:
ok. Then the short printing will go into Base.show and the long one into show_details. Does this suit you?

Reply:
Yes. Thank you. Though `IdealSheaf` already has a print method. Did you delete the old one?

Reply:
I just moved it to a comment. (If Matthias would decide to revert this change, I would not object) 
My idea is that Cartier (and later also Weil) divisors print via base.show in a similar way as IdealSheafs and that the user gets some information on the size of the covering of the underlying scheme.